underscore.js

```diff
@@ -1446,6 +1453,13 @@
     return _.isFunction(value) ? value.call(object) : value;
   };
 
+  // Returns the, possibly deeply nested, value at `path` if defined.
+  // Otherwise, returns `fallback`.
+  _.get = function(object, path, fallback) {
+    var value = property(path)(object);
+    return value === void 0 ? fallback : value;
+  };
```

Comment:
Hmm. Maybe I did something strange? I believe I posted (nearly) the same content in the commit and the pull request message. Here's what I see:

<img width="1054" alt="screen shot 2016-03-30 at 8 26 14 am" src="https://cloud.githubusercontent.com/assets/162735/14147622/1cadab4c-f651-11e5-8331-e48aeb38938d.png">

If that's not what you're seeing, let me know and I can try opening a fresh PR.